jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Sorry, I still have two questions:

1. What is env settings?

models, names, and objects are not env?
https://github.com/start-jsk/jsk_apc/tree/master/jsk_arc2017_common/data
objects are also not changed so often.

2. What about placing `data/objects/cardboard_box`?

What is difference between the cardboard_box and other objects? Objects also have the info of dimensions.